docs/configuration.md

```diff
@@ -18,6 +18,8 @@
   - `enable_unsafe_install` - set to `true` to enable the use of the `--git-url` and `--zip-file` flags with
     [`arduino-cli lib install`][arduino cli lib install]. These are considered "unsafe" installation methods because
     they allow installing files that have not passed through the Library Manager submission process.
+- `locale` - the language used by Arduino CLI, the parameter is the language identifier in the standard format
+  `<language>_<COUNTRY>.<encoding>` (for example `it` or `it_IT`, or `it_IT.UTF-8`).
```

Comment:
```suggestion
- `locale` - the language used by Arduino CLI to communicate to the user, the parameter is the language identifier in
  the standard format `<language>_<COUNTRY>.<encoding>` (for example `it` or `it_IT`, or `it_IT.UTF-8`).
```

The term "language" is quite overloaded in this sphere so I think it is worth being a little more explicit about exactly what it means in this context.